src/main/java/sc/iview/commands/demo/advanced/ShowSegmentationDemo.java

```diff
@@ -30,6 +30,7 @@
 
 import graphics.scenery.volumes.Volume;
 import io.scif.services.DatasetIOService;
+import kotlin.Unit;
 import net.imagej.mesh.Mesh;
```

Comment:
This is likely not necessary, is it?

Reply:
Ah, yes. I'd started out adding a keyword to `SciView.addNode` but that wasn't necessary.